SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+If you have discovered a security vulnerability in this project, please report it
+privately. **Do not disclose it as a public issue.** This gives us time to work with you
+to fix the issue before public exposure, reducing the chance that the exploit will be
+used before a patch is released.
+
+You may submit the report in the following ways:
+
+- send an email to ???@???; and/or
```

Comment:
We don't have an appropriate email alias. We can create one, but it seems better to set us up to receive private vulnerability reports.

Reply:
+1. I enabled the private vulnerability report feature.